tests/test_appliancesmanager.py

```diff
@@ -1,123 +1,101 @@
-import asyncio
+import json
+from pathlib import Path
 
 import pytest
 
 from whirlpool.appliancesmanager import AppliancesManager
-from whirlpool.auth import Auth
-
-from .aiohttp import AiohttpClientMocker
-from .mock_backendselector import BackendSelectorMock
-from .utils import (
-    ACCOUNT_ID,
-    get_mock_coro,
-    mock_appliancesmanager_get_account_id_get,
-    mock_appliancesmanager_get_owned_appliances_get,
-    mock_appliancesmanager_get_shared_appliances_get,
-)
-
-BACKEND_SELECTOR_MOCK = BackendSelectorMock()
-
-
-def assert_appliances_manager_call(
-    http_client_mock: AiohttpClientMocker,
-    call_index: int,
-    path: str,
-    required_headers: dict = None,
-    forbidden_headers: list = None,
-):
-    mock_calls = http_client_mock.mock_calls
+from whirlpool.backendselector import BackendSelector
 
-    call = mock_calls[call_index]
-    assert call[0] == "GET"
-    assert BACKEND_SELECTOR_MOCK.base_url + call[1].path == path
-    # call[2] is body, which will be None
+CURR_DIR = Path(__file__).parent
+DATA_DIR = CURR_DIR / "data"
+ACCOUNT_ID = "12345"
 
-    if required_headers is not None:
-        for k, v in required_headers.items():
-            assert call[3][k] == v
 
-    if forbidden_headers is not None:
-        for k in forbidden_headers:
-            assert k not in call[3]
+@pytest.fixture(autouse=True)
+def account_id_calls_fixture(aioresponses_mock, backend_selector_mock):
```

Comment:
```suggestion
def common_http_calls_fixture(aioresponses_mock, backend_selector_mock):
```
I was going to suggest `user_details_and_appliances_calls_fixture`, but it was a bit too big...